services/wallet/controllers_v3.go

```diff
@@ -252,15 +252,14 @@ func LinkZebPayDepositAccountV3(s *Service) func(w http.ResponseWriter, r *http.
 			})
 		}
 
-		xalr := &ZebPayLinkingRequest{}
-		if err := inputs.DecodeAndValidateReader(ctx, xalr, r.Body); err != nil {
+		zplReq := &ZebPayLinkingRequest{}
```

Comment:
I now recall where this variable's original name came from – the type, I believe, used to be called `XyzAbcLinkingRequest`, and the variable reflected that. When the type had been renamed, the variable had been left with the old name.